.github/workflows/deploy.yaml

```diff
@@ -1,7 +1,13 @@
-name: CI
+name: Deploy
 
 on:
-  workflow_dispatch: {}
+  workflow_dispatch:
+    inputs:
+      gitRef:
```

Comment:
Does this override the branch/tag dropdown that appears by default, or will it appear in addition to it?

Reply:
I think I know what you mean - appears in addition to it. 
![image](https://user-images.githubusercontent.com/11051676/165974682-9241af6a-21d5-45f8-bb9e-9cdb66cd6021.png)

It's somewhat confusing the "Use workflow from" dropdown is which branch to use for the workflow definition - which shouldn't change.

Reply:
Ohh of course - I hadn't spotted that 🤦 thanks!